src/hidimstat/base_variable_importance.py

```diff
@@ -117,17 +129,59 @@ def selection(
         else:
             mask_threshold_pvalue = np.ones(self.importances_.shape, dtype=bool)
 
-        self.selections_ = (
+        selections = (
```

Comment:
Should we add a test at the beginning to make sure that only one of the k_best, percentile ... is provided to avoid users who pass multiple arguments without knowing the internal prioritization?

Reply:
Example is selecting the 10-best variables and only the significant (higher than a threshold).

Reply:
the result with the 10 first which have a pvalue or an importance higher than 0.9 for example. In this case, it can 0 to 10 selected features.

Reply:
Ok, let's keep it this way,